systems/ds-stko-dev.json

```diff
@@ -31,4 +31,4 @@
     "jobCapabilities": [],
     "tags": [],
     "notes": {}
-}
+}
```

Comment:
This file has conflicts according to PR.

Reply:
Very strange, I'll resolve it here in a second.